### ISSUE TITLE
Failure of test "test_workflows.py" check WES4 model

### ISSUE THREAD

**Issue by lenastroeer** [virocon-organization/virocon#140]
**I'm submitting a**
  - bug report

## Actual behavior
After error correction of the references in intervals.py, the test "test_workflows.py" fails. We suspect that the test is faulty. The WES4 model would have to be checked again manually.

**Screenshot**

<img width="829" alt="Issue_test_WES4" src="https://user-images.githubusercontent.com/57865209/138650628-391dc704-ce15-4953-aeae-32ffafc41156.png">

**Comment by ahaselsteiner** [comment]
@lenastroeer although we fixed a bug in the PR#139 this problem was semi solved by not using the automatic test anymore, is that correct? Should the issue be kept open or closed?

**Comment by lenastroeer** [comment]
Yes, that is correct, we need to review test_workflows.py, so the issue should be kept open.
